webserver/preferences.js

```diff
@@ -619,3 +619,85 @@ document.addEventListener("DOMContentLoaded", () => {
         });
     }
 });
+
+/**
+ * Sanity Data form before Save 
+ */
+function sanityData() {
+    var txt_error = "";
+    let errorMessage = document.getElementById('error-message');
+    const displayB = document.getElementById("DisplayBright");
+    if (parseInt(displayB.value) > displayB.max || parseInt(displayB.value) < displayB.min) {
+        displayB.classList.remove('valid');
+        displayB.classList.add('error');
+
+        if (parseInt(displayB.value) > displayB.max) displayB.value = displayB.max;
+        if (parseInt(displayB.value) < displayB.min) displayB.value = displayB.min;
+
+        txt_error = "Display Brightness value must be >=" + displayB.min + " and <=" + displayB.max;
+        if (!errorMessage) {
+            errorMessage = document.createElement('div');
+            errorMessage.id = 'error-message';
+            errorMessage.className = 'form-error';
+            errorMessage.textContent = txt_error;
+            displayB.insertAdjacentElement('afterend', errorMessage);
+        }
+        return false;
+    } else {
+        if (errorMessage) errorMessage.remove();
+        displayB.classList.remove('error');
+        displayB.classList.add('valid');
+    }
+
+    const co2O = document.getElementById("co2OrangeRange");
+    const co2R = document.getElementById("co2RedRange");
```

Comment:
Please avoid using cryptic names for variables such as co2O or co2R. Variable names should be self-explanatory.